config/jobs/kubernetes-sigs/kind/kind.yaml

```diff
@@ -20,11 +19,11 @@ periodics:
       # we need privileged mode in order to do docker in docker
       securityContext:
         privileged: true
-    annotations:
-      testgrid-dashboards: sig-testing-kind
-      testgrid-tab-name: ci unit test
-      testgrid-alert-email: bentheelder@google.com
-      description: kind CI unit test
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: kind-ci-unit-test
```

Comment:
what has this changed?

Reply:
Kindly reference the [issue](https://github.com/kubernetes/test-infra/issues/27454) linked above for the diff and details as to why

Reply:
TLDR indentation was changed...

Reply:
no need

Reply:
I meant the name , it was `ci unit test`